sops/__init__.py

```diff
@@ -1278,10 +1278,13 @@ def write_file(tree, path=None, filetype=None):
         fd = tempfile.NamedTemporaryFile(suffix="."+filetype, delete=False)
         path = fd.name
 
-    if fd and not isinstance(tree, dict) and not isinstance(tree, list):
```

Comment:
Wouldn't it be simpler to just change line 1276 from `fd = None` to `fd = sys.stdout`?